DEPLOYING.md

```diff
@@ -69,52 +69,44 @@ contains step-by-step deployment instructions using opeNode's command line inter
 This same process is outlined in the next section.
 
 #### Deploy your app to your new instance
-1. On your local machine, open a command prompt.
+Note: A video walkthrough can be found here to supplment the instructions below: https://www.youtube.com/watch?v=SRzQOEwR4dM&list=PLmFHH6TuGBX1GjMAfzwz1wYz274RHq4ml&index=7&t=0s 
```

Comment:
Did you intend to link to the whole playlist? Either way I approve, great resources!

Could you format this as a markdown link instead of URL text? `Note: A [video walkthrough](url) is available...`

Reply:
Oh fantastic yes the only reason I didn't link it was not sure if it was appropriate